grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy

```diff
@@ -82,8 +82,19 @@ class InvoiceItem implements Serializable {
         product(nullable: true)
         glAccount(nullable: true)
         budgetCode(nullable: true)
-
-        quantity(nullable: false, min: 0) // min = 0 for canceled items
+        quantity(nullable: false, min: 0, validator: { Integer quantity, InvoiceItem obj ->
+                Integer notUpdatedQuantity = obj.getPersistentValue('quantity')
```

Comment:
We still need to come up with a better name for this variable.
`quantityInvoicedOriginal` or `originalQuantityInvoiced` would be my vote. Or even `previousQuantityInvoiced`, `oldQuantityInvoiced` or `savedQuantityInvoiced` would suffice.